src/app/static/src/app/components/adr/SelectDataset.vue

```diff
@@ -158,15 +168,18 @@
                 return outOfDateResources
             },
             outOfDateMessage() {
+                const updatedNames = Object.keys(this.outOfDateResources)
+                    .map(r => names[r as keyof DatasetResourceSet]).join(", ")
                 if (Object.keys(this.outOfDateResources).length == 0) {
                     return ""
                 }
-                return "This dataset has been updated in the ADR. Use the refresh button to import the latest files."
+                return `The following files have been updated in the ADR: ${updatedNames}. Use the refresh button to import the latest files.`
```

Comment:
This more informative msg was requested by Jeff. One weird thing though is that the ADR gives false positives - sometimes changing 1 file updates the metadata for other files in the set, so it will report that they've all changed when they haven't. If this causes confusion we'll have to revert to the simpler message.

Reply:
Are ADR aware of this issue?

Reply:
Yep.